TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/POST-Positive-Deviceresource.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Resource     TAF/testCaseModules/keywords/common/commonKeywords.robot
+Resource     TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
+Suite Setup  Run Keywords  Setup Suite
+...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
+Suite Teardown  Run Teardown Keywords
+Force Tags      Skipped
+
+*** Variables ***
+${SUITE}          Core Metadata Device Profile POST Resource Positive Test Cases
+${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-deviceprofile-post-deviceresource-positive.log
+
+*** Test Cases ***
+ProfileResourcePOST001 - Add deviceResource on one device profile basicinfo only
+    # one resource > one basicinfo profile
+    Given Create A Device Profile Basicinfo Only
+    When Add A New resource on device profile Basicinfo Only
+    Then Should Return Status Code "207"
+    And Should Return Content-Type "application/json"
+    And Item Index All Should Contain Status Code "201" And id
```

Comment:
The response doesn't return id. 
And add a check to verify the new device resource is added.